v2/swift/main.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  main.swift
+//  YelpAPISample
+//
+
+import Foundation
+
+// Fill in your API keys here from
+// https://www.yelp.com/developers/manage_api_keys
+let client = YLPClient(consumerKey: "",
+                       consumerSecret: "",
+                       token: "",
+                       tokenSecret: "")
+
+// Create a dispatch group to wait for search results
+let group = dispatch_group_create()
+dispatch_group_enter(group)
```

Comment:
We don't _really_ need a `dispatch_group` here since we're not trying to synchronize multiple blocks. From what it looks like, we just don't want to exit the current scope before our search result gets back, correct?

How would you feel about using a semaphore instead? Its a simpler construct and _maybe_ (emphasis on maybe) it might have less conceptual overhead. I don't feel very strongly about this though, so I'll leave it up to you.